src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -61,6 +88,15 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
     Utils utils;
     vector<Utils::worker> workerList = utils.getWorkerList(sqlite);
     vector<DataPublisher*> workerClients;
+//  Initiate Kafka consumer
+    thread th1;
```

Comment:
Rather than having the variable name `th1` lets have more meaningful variable name here.

Reply:
Ok sir